package.json

```diff
@@ -24,24 +24,24 @@
   "homepage": "https://github.com/amannn/action-semantic-pull-request#readme",
   "dependencies": {
     "@actions/core": "^1.10.0",
-    "@actions/github": "^5.0.0",
-    "conventional-changelog-conventionalcommits": "4.5.0",
+    "@actions/github": "^5.1.1",
+    "conventional-changelog-conventionalcommits": "5.0.0",
     "conventional-commit-types": "^3.0.0",
     "conventional-commits-parser": "^3.2.4"
   },
   "devDependencies": {
-    "@semantic-release/changelog": "5.0.1",
-    "@semantic-release/commit-analyzer": "8.0.1",
-    "@semantic-release/git": "9.0.0",
-    "@semantic-release/github": "7.2.0",
-    "@semantic-release/release-notes-generator": "9.0.3",
-    "@vercel/ncc": "^0.27.0",
-    "eslint": "7.9.0",
-    "eslint-config-molindo": "5.1.1",
-    "jest": "27.4.7",
-    "semantic-release": "19.0.2"
+    "@semantic-release/changelog": "6.0.2",
+    "@semantic-release/commit-analyzer": "9.0.2",
+    "@semantic-release/git": "10.0.1",
+    "@semantic-release/github": "8.0.7",
+    "@semantic-release/release-notes-generator": "10.0.3",
+    "@vercel/ncc": "^0.36.1",
+    "eslint": "8.36.0",
+    "eslint-config-molindo": "6.0.0",
+    "jest": "29.5.0",
+    "semantic-release": "20.1.1"
```

Comment:
Is there anything relevant for this repo to consider with the new major versions of the dependencies?

Reply:
My train of thought was that these releases will have longer support. If there would be CWE issues in dependencies of the underlying packages, they'd be resolved more quickly in the current version of the package itself, rather than updating the yarn.lock file.
If you would prefer to only update to the minor version increase, I can look into that too.

Reply:
Oh no, updating the major version for these dependencies is definitely fine! I was just curious if you've checked the changelogs of the dependencies to see if we need to update anything?

Reply:
## `Jest`
If there would be breaking changes, the tests will fail. I updated to Jest 29 with other branches where I needed to restore those. As there are no changes here, I don't see any problems there.

## `@semantic-release/changelog`
### BREAKING CHANGES
node-version: the minimum required version of node is now v14.17
That's already covered

## `@semantic-release/commit-analyser`
### BREAKING CHANGES
the minimum required version of node is now v14.17

## `@semantic-release/git`
### BREAKING CHANGES
node-version: the minimum required version of node is now v14.17

Reply:
Looks great, thanks for the details!